client/components/event-details/SimilarEvents.tsx

```diff
@@ -43,11 +44,29 @@ export const SimilarEvents = ({ allEvents, event }: SimilarEventsProps) => {
   )
 }
 
+const EventCardWrapper = styled.div`
+  cursor: pointer;
+`
+
 const EventCards = ({ events }: { events: Array<Event> }) => {
+  const router = useRouter()
+
+  const handleClick = (id: Id) => {
+    router.push({ pathname: `/events/${id}` })
+  }
+
   return (
     <div>
       {events.map((e, i) => {
```

Comment:
`event` is probably a better name, since `e` is usually used for a different kind of event